x/gov/client/cli/parse.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/spf13/viper"
 
-	govClientUtils "github.com/cosmos/cosmos-sdk/x/gov/client/utils"
+	govutils "github.com/cosmos/cosmos-sdk/x/gov/client/utils"
```

Comment:
`govutils` vs `clientutils` or the unfortunate `govclientutils`?

Reply:
I will rename to `utils` there is no need for an alias at all